### PR TITLE
Add ability to select integration tests and change log level

### PR DESCRIPTION
**Pull request by dmage**
Usage example:

    LOGLEVEL=5 ./hack/test-integration.sh -run TestIntegration/TestImageStreamImportDockerHub -v

### CODE REVIEW

**Comment by mfojtik** [comment]
\o/

@stevekuznetsov any objections?

**Comment by stevekuznetsov** [comment]
Please accompany this with an update to `HACKING.md` 

/cc @enj

**Comment by enj** [comment]
Added do not merge, this is incorrect, will update later when I have time.

**Comment by enj** [comment]
xref #16153

**Comment by stevekuznetsov** [comment]
@enj we have a process, gosh darn it

**Comment by dmage** [comment]
Ok, lets focus our efforts in that PR. I have two bugs there: GOTESTFLAGS is ignored, the verbosity level is always set to the maximum value.

File: test/integration/runner/runner_test.go [hunk 1]
```diff
@@ -157,6 +158,7 @@ func runSingleTest(t *testing.T, dir, binaryPath, name string) error {
 		cmd.Args = append(cmd.Args, "-test.short")
 	}
 	cmd.Args = append(cmd.Args, "-test.timeout", (*timeout).String())
+	cmd.Args = append(cmd.Args, "-v", fmt.Sprintf("%d", loglevel))
```
Comment (dmage) [inline test/integration/runner/runner_test.go]: BUG: `loglevel` -> `*loglevel`

**Pull request closed without merging**